_episodes/03-running-containers.md

```diff
@@ -12,21 +12,21 @@ objectives:
 - "Understand container state"
 - "Stop and restart containers"
 keypoints:
-- "Run containers with `docker run <image-id>`"
-- "Monitor containers with `docker ps`"
+- "Run containers with `podman run <image-id>`"
+- "Monitor containers with `podman ps`"
 - "Exit interactive sessions using the `exit` command"
-- "Restart stopped containers with `docker start`"
+- "Restart stopped containers with `podman start`"
```

Comment:
again, I don't see "key points" on the web page...

Reply:
<img width="1199" alt="image" src="https://github.com/hsf-training/hsf-training-docker/assets/2147367/19a6c7fd-af6d-4eed-8030-bb14ee675667">
Strange... they are at the bottom, could you check again?

Reply:
yes, again, sorry for missing it